src/ashell/jerry-code.c

```diff
@@ -74,6 +134,123 @@ static void javascript_print_value(const jerry_value_t value)
     jerry_port_console("\n");
 }
 
+static char *read_file(const char *file_name, ssize_t *size)
```

Comment:
Add alloc to the name or something, to make it clear that it's returning a new buffer. Otherwise, it's a memory leak waiting to happen when Mr. Maintenance comes along.

E.g. `read_alloc_file`